c2corg_ui/templates/waypoint/edit.html

```diff
@@ -7,139 +9,774 @@
+            <div class="help-block" ng-messages="editForm.waypoint_type.$error">
+              <p ng-message="required"  style="{{ (editForm.waypoint_type.$invalid && editForm.waypoint_type.$touched) ? 'display:inline' : ''}}" translate>This field is required.</p>
+            </div>
+          </div>
+
+          ## RAIN PROOF
+          <div class="form-group data full" ng-if="waypoint.waypoint_type == 'climbing_outdoor'" ng-class="{ 'has-error': editForm.rain_proof.$touched && editForm.rain_proof.$invalid, 'has-success': editForm.rain_proof.$valid }">
+            <label><span translate>rain_proof</span></label>
+            <select class="form-control " ng-model="waypoint.rain_proof" ng-options="type as mainCtrl.translate(type) for type in ${rain_proof_types}"></select>
+            <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="waypoint.rain_proof"></span>
+          </div>
+
+          ## CHILDREN PROOF
+          <div class="form-group data full" ng-if="waypoint.waypoint_type == 'climbing_outdoor'">
+            <label translate>children_proof</label>
+            <select ng-options="type as type for type in ${children_proof_types}" ng-model="waypoint.children_proof" class="form-control "></select>
+            <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="waypoint.children_proof"></span>
+          </div>
+
+        </div>
+
+        ## ORIENTATION
+        <div class="form-group data half"  ng-if="waypoint.waypoint_type == 'paragliding_takeoff' || waypoint.waypoint_type == 'paragliding_landing' || waypoint.waypoint_type == 'climbing_outdoor'"
+             ng-class="{ 'has-error': editForm.orientation.$touched && editForm.orientation.$invalid,'has-success': editForm.orientation.$valid }">
+          <label><span translate>orientation</span></label>
+          <select id="orientation-select" class="form-control" ng-options="or as or for or in ${orientation_types}" ng-model="waypoint.orientation"></select>
+          <%include file="c2corg_ui:static/img/orientation.svg" />
+          <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="editForm.orientation.$valid"></span>
+        </div>
+
+        ## WEATHER STATION TYPES
+        <div class="form-group data half" ng-if="waypoint.waypoint_type == 'weather_station'"
+             ng-class="{ 'has-error': editForm.weather_station_types.$touched && editForm.weather_station_types.$invalid, 'has-success': editForm.weather_station_types.$valid }">
+          <label><span translate>weather_station_types</span></label>
+          <ul class="types">
+            <li ng-repeat="type in ${weather_station_types}"  ng-click="editCtrl.pushToArray(waypoint, 'weather_station_types', type, $event)">
+              <div class="checkbox"><input type="checkbox" ng-checked="waypoint.weather_station_types.indexOf(type) > - 1"><span translate>{{type}}</span></div>
```

Comment:
Please use ``x-translate`` here instead of ``translate``.
Ditto for translating the other ``{{type}}`` occurences.